### DISCUSSION TITLE
How to troubleshoot this?

### DISCUSSION THREAD

**Discussion by bcsteeve** [immich-app/immich#3468] [General]
Keeping in mind that I have no idea where logs are, etc... how do I troubleshoot this issue:

![image](https://github.com/immich-app/immich/assets/7412219/e608aeaf-f72d-4dd7-a5dc-3fbcbbdcbfb1)

Basically, any time I favorite something, or unfavorite, or delete anything I get an error like this.  But the thing I was trying to do does happen (ie. it is favorited or unfavorited or deleted despite the error message).

I'd like to say this started happening a few updates ago.  Hasn't always been like this.

Thanks.

**Comment by alextran1502** [comment]
Is this the asset that you own, or is shared with you?

**Reply by bcsteeve** [reply]
I'm not using sharing at all.   I own it.  Thanks.

**Reply by alextran1502** [reply]
This error popup when you favorite and then unfavorite, correct?

**Reply by alextran1502** [reply]
Can you open the console panel (F12) and look at the console tab and capture the screenshot of the error message when this happens

**Reply by bcsteeve** [reply]
Either favorite OR unfavorite OR delete (probably other actions too).

I'll do as you say and post shortly.

**Reply by bcsteeve** [reply]
Is this what you wanted?  

![image](https://github.com/immich-app/immich/assets/7412219/866c2a8c-a03f-4d44-8ffa-ef233e6209a8)

ps. I noticed if I favorite/unfavorite from the main photos page, it doesn't behave like this.  But if I do it within the Favorites "folder" (for lack of a better term), then it does.

**Reply by alextran1502** [reply]
Thank you, I will investigate

**Reply by alextran1502** [reply]
FYI, for future reference when you encounter issue like this, please feel free to open an Issue instead of the discussion so we can track it 😃

**Reply by bcsteeve** [reply]
I wanted to investigate it a bit before unnecessarily sounding the alarm on a bug that might be just a permissions issue with my setup :)

**Reply by alextran1502** [reply]
I can reproduce this. I think I know where the issue is. Thank you
